[2025-05-21T16:19:30.374+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-21T16:19:30.403+0000] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_age scheduled__2024-12-15T00:00:00+00:00 [queued]>
[2025-05-21T16:19:30.421+0000] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_age scheduled__2024-12-15T00:00:00+00:00 [queued]>
[2025-05-21T16:19:30.428+0000] {taskinstance.py:2884} INFO - Starting attempt 1 of 6
[2025-05-21T16:19:30.461+0000] {taskinstance.py:2907} INFO - Executing <Task(_PythonDecoratedOperator): get_age> on 2024-12-15 00:00:00+00:00
[2025-05-21T16:19:30.472+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=10630) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-21T16:19:30.494+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'get_age', 'scheduled__2024-12-15T00:00:00+00:00', '--job-id', '3441', '--raw', '--subdir', 'DAGS_FOLDER/dar_with_taskflow_api.py', '--cfg-path', '/tmp/tmp5ob4cua7']
[2025-05-21T16:19:30.507+0000] {standard_task_runner.py:72} INFO - Started process 10655 to run task
[2025-05-21T16:19:30.507+0000] {standard_task_runner.py:105} INFO - Job 3441: Subtask get_age
[2025-05-21T16:19:30.667+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_api_v02.get_age scheduled__2024-12-15T00:00:00+00:00 [running]> on host 2032228d86df
[2025-05-21T16:19:31.214+0000] {taskinstance.py:3157} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v02' AIRFLOW_CTX_TASK_ID='get_age' AIRFLOW_CTX_EXECUTION_DATE='2024-12-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-15T00:00:00+00:00'
[2025-05-21T16:19:31.228+0000] {taskinstance.py:740} INFO - ::endgroup::
[2025-05-21T16:19:31.247+0000] {python.py:240} INFO - Done. Returned value was: 19
[2025-05-21T16:19:31.497+0000] {taskinstance.py:349} INFO - ::group::Post task execution logs
[2025-05-21T16:19:31.506+0000] {taskinstance.py:361} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v02, task_id=get_age, run_id=scheduled__2024-12-15T00:00:00+00:00, execution_date=20241215T000000, start_date=20250521T161930, end_date=20250521T161931
[2025-05-21T16:19:31.630+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-21T16:19:31.769+0000] {taskinstance.py:3924} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-21T16:19:31.798+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
